pkg/beacon/dkg/dkg.go

```diff
@@ -216,8 +217,10 @@ func waitForDkgResultEvent(
 func resolveGroupOperators(
 	selectedOperators []chain.Address,
 	operatingGroupMembersIDs []group.MemberIndex,
+	beaconConfig *beaconchain.Config,
 ) []chain.Address {
-	if len(selectedOperators) == 0 || len(operatingGroupMembersIDs) == 0 {
+	if len(selectedOperators) != beaconConfig.GroupSize ||
+		len(operatingGroupMembersIDs) < beaconConfig.HonestThreshold {
 		return []chain.Address{}
```

Comment:
Shouldn't we return an error instead? Empty array of group operators is not a valid state.

Reply:
See c8463717